gen/dcos-config.yaml

```diff
@@ -119,6 +119,7 @@ package:
         {dcos_l4lb,
           [
             {enable_lb, {{ enable_lb }}},
+            {enable_ipv6, false},
```

Comment:
@urbanserj we definitely need to update the documentation with these options. Could you kindly create a JIRA for this?

Reply:
I've added a comment to the existing jira ticket: https://jira.mesosphere.com/browse/DCOS_OSS-1404?focusedCommentId=131849&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-131849